apps/financial-aid/api/src/environments/environment.ts

```diff
@@ -36,7 +36,7 @@ const prodConfig = {
 const devConfig = {
   production: false,
   auth: {
-    samlEntryPoint: 'https://innskraning.island.is/?id=financial-aid.local',
+    samlEntryPoint: 'https://innskraning.island.is/?id=judicial-system.local',
```

Comment:
Why are you doing this?

Reply:
financial-aid is not in the login system yet, so I'm piggy backing on the judicial-system